### ISSUE TITLE
add pokemon whitelist to config

### ISSUE THREAD

**Issue by Courageousodin** [timinc-cobble/cobblemon-spawnnotification#35]
hey, I wanted to see if I could add starter pokemon to the spawn notification since they are so rare and sought after. I saw you can add pokemon by labels but it seems the starter pokemon only have labels for their generation. Was wondering if you could add a feature to add different pokemon to the config so that alerts would be played for them. 

Thanks for the cool mod!

**Comment by Mortycio** [comment]
up

**Comment by tmetcalfe89** [comment]
Yes, the ability to add and broadcast for custom labels has always been supported by the mod. Detailed information on how can be found on the mod's Discord.
